### PR TITLE
`Node.ownerDocument` has been supported since IE 6

### PR DESCRIPTION
**Pull request by aleen42**
A checklist to help your pull request get merged faster:

- [x] Summarize your changes

    Update support of `Node.ownerDocument` under IE8

- [ ] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)
- [x] Data: if you tested something, describe how you tested with details like browser and version

    I have tested under IE 6 and 8, and I found that the following snippet worked:

    ```js
    console.log(document.body.ownerDocument === document); // => true
    console.log(document.querySelector('*').ownerDocument === document); // => true
    ``` 

- [x] Review the results of the linter and fix problems reported (If you need help, please ask in a comment!)
- [ ] Link to related issues or pull requests if any

### CODE REVIEW

**Comment by foolip** [comment]
@vinyldarkscratch can you review this against collector results? I suspect this attribute may have been on the wrong prototype in IE.

**Comment by aleen42** [comment]
any progress?

File: api/Node.json [hunk 1]
```diff
@@ -1699,7 +1699,7 @@
               }
             ],
             "ie": {
-              "version_added": "9"
+              "version_added": "8"
```
Comment (vinyldarkscratch) [inline api/Node.json:1702]: It looks like this was actually supported much earlier than IE 8.  Using the [mdn-bcd-collector project](https://mdn-bcd-collector.appspot.com/tests/api/Node/ownerDocument) and BrowserStack, I was able to confirm support all the way back to IE 6, possibly further.  Can we set the version accordingly?

```suggestion
              "version_added": "≤6"
```

**Review by vinyldarkscratch — CHANGES_REQUESTED** [review]

File: api/Node.json (hunk 1 above)
Comment (aleen42) [reply api/Node.json:1702]: It is safe after tested under my IE6

image

**Comment by vinyldarkscratch** [comment]
I saw you set the version to IE 5.5, how did you get that version number?  Did you test in IE 5 and 5.5 as well, or find it in an article somewhere?

**Comment by aleen42** [comment]
@vinyldarkscratch Sorry for the mistake

> IHTMLDOMAttribute2::ownerDocument was introduced in Microsoft Internet Explorer 6.

https://docs.microsoft.com/en-us/previous-versions/windows/internet-explorer/ie-developer/platform-apis/jj160764(v=vs.85)

**Review by vinyldarkscratch — APPROVED** [review]
Awesome, thank you for confirming!  With that article, we can confidently set this straight to IE 6, I'd say.  Let's get this merged!

P.S. there's no need to force push and squash your changes, as this repo strictly squash merges pull requests.  😉

**Comment by aleen42** [comment]
Squashing the commit branch is just my habit to prettify my local branch. Thanks for your cooperation.

**Pull request merged**